src/index.css

```diff
@@ -1,6 +1,25 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,100;1,300;1,400;1,500;1,700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+*::selection {
+  background-color: #ff3f67;
+  color: #ddd;
+}
+
+html {
+  overflow-y: hidden;
+}
```

Comment:
- [ ] Kindly check the height of your calculator, as some part of the calculator is not visible because of this property.
![Capture](https://user-images.githubusercontent.com/85095702/171235074-bae2a294-e6f0-4806-a13b-30b2bff01f60.jpg)